lib/ansible/modules/files/jsonmod.py

```diff
@@ -0,0 +1,382 @@
+#!/usr/bin/python -tt
```

Comment:
Shebang line should be `#!/usr/bin/python` as per [Contributing Modules Checklist](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_checklist.html#contributing-modules-checklist)